content/en/docs/tasks/debug-application-cluster/crictl.md

```diff
@@ -3,67 +3,61 @@ reviewers:
 - Random-Liu
 - feiskyer
 - mrunalp
-title: Inspect Kubernetes node with crictl
+title: Debugging Kubernetes nodes with crictl
+content_template: templates/task
 ---
 
-crictl provides a CLI for CRI-compatible container runtimes. It helps users inspect and debug container runtime and applications on a Kubernetes node.
+{{< toc >}}
 
-crictl is GA since v1.11.0 and is hosted at the [cri-tools](https://github.com/kubernetes-incubator/cri-tools) repository.
+{{% capture overview %}}
 
-{{< toc >}}
+{{< feature-state for_k8s_version="v1.11" state="stable" >}}
 
-## Install crictl
+`crictl` is a command-line interface for CRI-compatible container runtimes.
+You can use it to inspect and debug container runtimes and applications on a
+Kubernetes node. `crictl` and its source are hosted in the
+[cri-tools](https://github.com/kubernetes-incubator/cri-tools) repository.
 
-crictl can be downloaded from cri-tools [release page](https://github.com/kubernetes-incubator/cri-tools/releases):
+{{% /capture %}}
 
-```sh
-VERSION="v1.11.0"
-wget https://github.com/kubernetes-incubator/cri-tools/releases/download/$VERSION/crictl-$VERSION-linux-amd64.tar.gz
-sudo tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /usr/local/bin
-rm -f crictl-$VERSION-linux-amd64.tar.gz
-```
+{{% capture prerequisites %}}
 
-## Usage
+In order to use `crictl`, you need access to a Linux system that is not a
+running Kubernetes cluster. 
```

Comment:
crictl could be running with Kubernetes cluster. It can help for troubleshooting runtime issues. 

The only limitation is that: if you create a container with crictl directly, the container will be stopped and removed by kubelet.